client/tokenized-express-checkout/index.js

```diff
@@ -374,8 +389,9 @@ jQuery( ( $ ) => {
 						}
 
 						expressCheckoutButtonUi.unblockButton();
+						expressCheckoutButtonUi.showContainer();
```

Comment:
Just in case it had been hidden before.

Reply:
I think [it's part of the `unlockButton`](https://github.com/Automattic/woocommerce-payments/blob/develop/client/tokenized-express-checkout/button-ui.js#L28-L31) - should we call it again here?

Reply:
You're right - thank you for pointing it out!
```suggestion
```